test/examples_test.go

```diff
@@ -239,7 +239,7 @@ func TestYamls(t *testing.T) {
 }
 
 func testYamls(t *testing.T, baseDir string, createFunc createFunc, filter pathFilter) {
-	t.Parallel()
+	// t.Parallel()
```

Comment:
(im assuming this is temporary for the experiment you're running?)

Reply:
Yeah, it's uncommented again now!